### PR TITLE
Manual publish

### PR DESCRIPTION
**Pull request by WickyNilliams**
### What is the context of this PR?
Doing some bits ahead of a manual publish to npm

### How to review 
Sanity check changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # eq-author-graphql-schema
 
 GraphQL type definitions and schema for [eq-author](https://github.com/ONSdigital/eq-author).
+
+## Publishing
+
+1. Create new branch
+2. Make any changes
+3. Commit changes
+4. Run `npm version [minor|patch|major]`. This will:
```
Comment (samiwel) [inline README.md]: Do we need to define `minor`, `patch` and `major` in the README for some guidance about which one to use?

**Comment by WickyNilliams** [comment]
[Improve versioning of schema (2)](https://trello.com/c/ssZ3xXxt/229-improve-versioning-of-schema-2)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    1. Update the package.json version
+    2. Commit changes to package.json
+    3. Create a git tag
+    4. Push changes
+5. Create PR
+6. Merge PR
+7. Pull master
+8. Run `npm publish` (ensure you have the correct credentials)
```
Comment (samiwel) [inline README.md]: Which credentials should we be using for publishing to npm?

Comment (WickyNilliams) [reply README.md]: Credentials for npm. Which of course I didn't want to add them to a public README :) A few of us have them, somebody can give them to you if you need them.

File: README.md (hunk 1 above)
Comment (WickyNilliams) [reply README.md]: As per semver http://semver.org/, which npm requires

**Pull request merged**